package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "npm run development",
+    "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "watch": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "watch-poll": "npm run watch -- --watch-poll",
+    "hot": "cross-env NODE_ENV=development node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "prod": "npm run production",
+    "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
+  },
+  "dependencies": {
+    "axios": "^0.17.1",
+    "jquery": "^3.1.0",
+    "vue": "^2.5.9",
+    "vue-awesome": "^2.3.4",
+    "vue-clipboard2": "0.0.8",
+    "vue-ls": "^2.3.3",
+    "vue-notif": "^1.1.0",
+    "vue-tippy": "^1.0.0",
+    "vuemit": "^1.0.7"
+  },
+  "devDependencies": {
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-stage-3": "^6.24.1",
+    "cross-env": "^5.0.1",
+    "es6-promise": "^4.0.5",
+    "laravel-mix": "^1.4.2"
+  }
+}
```

Comment:
again why do u need this ? other than restricting the packages versions

Reply:
You always need to restricted package version,  from v1 to v2 can break things

and again this for the developer